recipes/msaprobs/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: msaprobs
+  version: 0.9.7
+
+source:
+  url: https://sourceforge.net/projects/msaprobs/files/MSAProbs-0.9.7.tar.gz
+  sha256: a14c59d714a5020c091ba9dd64d57d4d4aa5e39fefec06ba2f3d29e9ab38dad0
+  patches:
+    - Makefile.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - zlib
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+  host:
+    - llvm-openmp  # [osx]
+    - openmp  # [osx]
+  run:
+    - llvm-openmp  # [osx]
+    - openmp  # [osx]
+
+test:
+  commands:
+    - msaprobs -version 2>&1 | grep "version 0.9.7"
+
+about:
+  home: http://msaprobs.sourceforge.net/homepage.htm
+  license: "General Public License v3.0"
+  author: Yongchao Liu
```

Comment:
The GPL 3 requires that a copy of the license is distributed along with the Conda package. Does the `.tar.gz` include a copy of the GPL? Then use `license_file: LICENSE.txt` (or whatever the filename) here.